### DISCUSSION TITLE
What's new in 0.3.9?

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#932] [Q&A]
@maluoi I see you pushed 0.3.9, what's new? Have a question. I'm trying to use a browser in an SK app, do you know how I can embed the Edge browser app into an SK app?

**Comment by maluoi** [comment]
Release notes are over here!
https://github.com/StereoKit/StereoKit/releases/tag/v0.3.9

There's a sample that shows embedding a Chromium browser over here, platform support may be a bit limited for this particular sample, but it should be quite similar to most other embedded browsers. I believe WebView2 may be available for use on UWP these days?
https://github.com/StereoKit/StereoKit-Browser

**Comment by mvvvv** [comment]
I tested this solution with rust on the quest2 and it works well: https://youtu.be/28H67HftB_E                                                                      
                       
I believe it could be easily adapt with xamarin : https://github.com/amodm/webbrowser-rs/blob/main/src/android.rs

it opens the browser from the app the same way you can open the browser using the Quest menu.

I will try to adapt this code to open all the Quest app that can be launch from an app.

**Comment by laultman** [comment]
Thanks!
